### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by hassaanQadir**
This is the rough draft of my 30Aug24 Fine-Tuning Llama2 Article.

### CODE REVIEW

File: finetuning/llama/README.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
+Let’s say you want to fine-tune llama3. By running PEFT LoRA fine-tuning on a dataset with a few thousand rows in it, you can dramatically increase the accuracy of your model’s responses, and if you use Beam for compute, you can accomplish it in 1 hour rather than 10.
```
Comment (mernit) [inline finetuning/llama/README.md:1]: Add a title

File: finetuning/llama/README.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+
+Go to beam.cloud and create a Pay-As-You-Go Developer account. You’ll receive the API key and instructions on how to save it to your environment. Follow these instructions.
+
+1. Access Llama3 weights
+
+Next, you need the weights of the Llama3 base model to begin fine-tuning them. Go to [llama.meta.com/llama-downloads/](https://llama.meta.com/llama-downloads/) to request access, which is typically granted immediately, along with a custom URL and instructions on using it to download the weights. Follow the instructions.
+
+1. Upload the weights and dataset to a Beam Volume
+
+Finally, upload the weights and your fine-tuning dataset to a Beam Volume using the following CLI commands.
```
Comment (mernit) [inline finetuning/llama/README.md:17]: Include a 1-2 sentence explanation of how to choose a suitable dataset for fine-tuning, you can also link to an external blog post or article

**Review by mernit — CHANGES_REQUESTED** [review]
This is a good distillation of the E2E workflow, my only concern is that we didn't demonstrate that the model is actually fine-tuned or doing anything different than the ordinary base model -- but this is probably fine for now

File: finetuning/llama/README.md (hunk 1 above)
Comment (hassaanQadir) [reply finetuning/llama/README.md:3]: I added # Fine-Tuning Llama3 with LoRA and Beam

**Review by hassaanQadir — COMMENTED** [review]
I added a title and some lines about datasets for fine-tuning, plus a link to example datasets and guidelines from meta.

**Pull request merged**